Source/HeartCore/Public/Input/HeartInputActivation.h

```diff
@@ -146,7 +146,7 @@ enum class EHeartInputActivationType : uint8
 };
 
 UCLASS()
-class UHeartInputActivationLibrary : public UBlueprintFunctionLibrary
+class HEARTCORE_API UHeartInputActivationLibrary : public UBlueprintFunctionLibrary
```

Comment:
Can you remove just this one change? Ill accept the rest, but I have a different idea for UHeartInputActivationLibrary

Reply:
Done